octopoes/octopoes/models/ooi/dns/records.py

```diff
@@ -155,3 +156,33 @@ class DNSPTRRecord(DNSRecord):
     @classmethod
     def format_reference_human_readable(cls, reference: Reference) -> str:
         return f"{reference.tokenized.address.address} -> {reference.tokenized.hostname.name}"
+
+
+class CAATAGS(Enum):
+    ISSUE = "issue"
+    ISSUEWILD = "issuewild"
+    IODEF = "iodef"
+    CONTACTEMAIL = "contactemail"
+    CONACTPHONE = "contactphone"
+    ISSUEVMC = "issuevmc"
+    ISSUEMAIL = "issuemail"
+    
+    def __str__(self):
+        return self.value
+
+
+class DNSCAARecord(DNSRecord):
+    object_type: Literal["DNSCAARecord"] = "DNSCAARecord"
+    dns_record_type: Literal["CAA"] = "CAA"
+
+    # https://datatracker.ietf.org/doc/html/rfc8659#name-canonical-presentation-form
+    # An unsigned integer between 0 and 255.
+    flags: Optional[int] = None
+
+    # A non-zero-length sequence of ASCII letters and numbers in lowercase.
+    tag: CAATAGS = None
```

Comment:
This field is unclear, is it required, or is it optional with a default, or is it a nullable field?
You probably want one of the following:
- `tags: CAATAGS`; this means required
- `tags: Optional[CAATAGS]`; this means required, but can be `None`
- `tags: Optional[CAATAGS] = None`; this means optional with a default value